app/templates/components/popover-message.hbs

```diff
@@ -0,0 +1,10 @@
+<i
+        class="fa fa-question-circle permission-info"
+        data-toggle="popover"
```

Comment:
Rather than bootstrap popovers, in preprints, we have been using a library called hint.css.
http://kushagragour.in/lab/hint/

(Less javascript is a good thing, and in the regular OSF we sometimes had weird UI glitches when we used the bootstrap variant)

Can we use hint.css here instead? If possible, we want to use only a single consistent way of showing tooltips so as to avoid stylistic and UI inconsistencies.